conf/i18n/constants.js

```diff
@@ -186,5 +186,28 @@ exports.SPEECH_RECOGNITION_LOCALES_SUPPORTED_BY_EDGE = [
   'de-at',
   'de-ch',
   'de-de',
-  'ja-jp'
+  'ja-jp',
+  'ar-ae',
+  'ar-bh',
+  'ar-dz',
+  'ar-iq',
+  'ar-kw',
+  'ar-ly',
+  'ar-om',
+  'ar-ps',
+  'ar-qa',
+  'ar-sa',
+  'nl-be',
+  'nl-nl',
+  'pt-br',
+  'pt-pt',
+  'sv-se',
+  'zh-hans-cn',
```

Comment:
just confirming 'zh-hans-sg` and 'zh-hant-mo' aren't supported?

Reply:
they are not, you can very quickly test them here https://codepen.io/oshi97/pen/RwVKKzB

Reply:
or by hacking the searchbar's SpeechRecognition instance through dev tools and ANSWERS.components